fdbclient/ServerKnobs.cpp

```diff
@@ -971,6 +971,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( BLOB_MANAGER_STATUS_EXP_BACKOFF_EXPONENT,              1.5 );
 	init( BLOB_MANAGER_CONCURRENT_MERGE_CHECKS,                   64 ); if( randomize && BUGGIFY ) BLOB_MANAGER_CONCURRENT_MERGE_CHECKS = 1 << deterministicRandom()->randomInt(0, 7);
 	init( BLOB_MANIFEST_BACKUP,                                false );
+	init( BLOB_MANIFEST_BACKUP_INTERVAL,   isSimulated ?  5.0 : 30.0 );
```

Comment:
30 seconds is likely still way too often for large clusters, but it's fine for now

Reply:
well noted. any suggestions for the interval? probably 10 mins or even longer?

Reply:
We had initially talked about on the order of several times a day, but that was also when the manifest flush included flushing all granules (which we still need to do to guarantee recent backups).
Just the manifest dump itself can be more frequent though, and is good to do more frequently. On the order of once every 10 minutes seems reasonable, but likely we'd just want to performance test it on a realistically large cluster to determine the overhead and time it takes to create it